tests/python/unittest/test_runtime_module_export.py

```diff
@@ -153,13 +158,13 @@ def verify_json_import_dso(obj_format):
             f.write(subgraph_json)
 
         # Get Json and module.
-        A = te.placeholder((1024,), name="A")
-        B = te.compute(A.shape, lambda *i: A(*i) + 1.0, name="B")
-        s = te.create_schedule(B.op)
-        f = tvm.build(s, [A, B], "llvm", name="myadd")
+        a = te.placeholder((1024,), name="a")
+        b = te.compute(a.shape, lambda *i: a(*i) + 1.0, name="b")
+        s = te.create_schedule(b.op)
+        f = tvm.build(s, [a, b], "llvm", name="myadd")
         try:
             ext_lib = tvm.runtime.load_module(subgraph_path, "examplejson")
-        except:
+        except:  # pylint: disable=bare-except
```

Comment:
possible to add at least Exception here?

Reply:
I didn't know what to add.